t/DbCheck.t

```diff
@@ -372,7 +372,7 @@ subtest 'Registry instantiation', sub {
 
   $check = TestChecks::DbCheck_1->new(
     dba        => $dba,
-    server_uri => $server_uri,
+    server_uri => [$server_uri],
```

Comment:
Is this turning initial input in an array? Should we consider renaming the variable to something plural? (it might be a bit dangerous though!)

Reply:
Yes, it's now an array - but I figured changing the name was more effort than it was worth. The scripts for running datachecks/pipelines, which are where most users will interact with the parameter, still accept string values, i.e. you use `--server_uri mysql://etc`, as before. So from a user perspective, it'd probably be unnecessarily disruptive to change the name.

Reply:
Fine, seeing the number of time this param is used, it's sensible, Thanks.